### ISSUE TITLE
🐛 Bug Report: Regression: videos no longer appear in posts

### ISSUE THREAD

**Issue by Daniel-Valentine** [libreddit/libreddit#658]
On Firefox 107.0.1, a post with an embedded video no longer shows the video when HLS playback is enabled. Example (`/r/Theranos/comments/qv7hpp/ill_never_get_over_this_moment_from_the/`):

![Screenshot 2022-12-06 at 00-48-42 I'll never get over this moment from the documentary  - r_Theranos](https://user-images.githubusercontent.com/13266270/205852337-8fda225d-4a5b-4db6-a1e0-dd499ee84781.png)

This appears to be unique to Firefox. I am not able to replicate this in Brave. I will try other Chromium browsers soon.

Git bisect reveals the bug first appeared in e62d33ccaed7742b3ed15bc5472fa9422f28d251, which introduced the feature to blur images/videos in NSFW posts (#636).

**Comment by Daniel-Valentine** [comment]
Appears to be an issue with, or at least related to, the `margin` property in rule `.post_media_content`.

```css
.post_media_content, .post .__NoScript_PlaceHolder__, .gallery {                                                                                   
    max-width: calc(100% - 40px);
    grid-area: post_media;
    margin: 15px auto 5px auto;
    width: auto;
    height: auto;
    overflow: hidden;
}
```

When I disable the `margin` property, the video controls reappear.

![Screenshot 2022-12-06 at 01-02-19 I'll never get over this moment from the documentary  - r_Theranos](https://user-images.githubusercontent.com/13266270/205854606-0ddf0281-60db-4caa-8d60-c3fb1a725b46.png)

**Comment by Daniel-Valentine** [comment]
This has been fixed in v0.25.1.
